overlord/ifacestate/helpers.go

```diff
@@ -98,6 +101,28 @@ func (m *InterfaceManager) addSnaps() error {
 	return nil
 }
 
+// fixDisconnectedCorePlugs will re-connect the network-bind-plug and
+// core-support-plug to the corresponding slots on the core snap.
+// This cures the effects of bug https://bugs.launchpad.net/snappy/+bug/1680097
+func (m *InterfaceManager) fixDisconnectedCorePlugs() error {
+	const coreName = "core"
+	for _, slotName := range []string{"network-bind", "core-support"} {
+		plugName := fmt.Sprintf("%s-plug", slotName)
```

Comment:
Pardon my ignorance - this means we need to modify the core snap to have a new `{network-bind,core-support}-plug` name, correct?